packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -590,10 +590,7 @@ class PageTransitionsTheme with Diagnosticable {
     Animation<double> secondaryAnimation,
     Widget child,
   ) {
-    TargetPlatform platform = Theme.of(context).platform;
-
-    if (CupertinoRouteTransitionMixin.isPopGestureInProgress(route))
```

Comment:
It seems like a weird way to override the platform. It may create unexpected behavior

Reply:
It's probably a legacy code.
So i deleted it😅